### ISSUE TITLE
Merge viewport-specific margin HTML divs with columns using offsets (col-*-push, col-*-pull)

### ISSUE THREAD

**Issue by oncletom** [GFDRR/open-risk-data-dashboard#201]
```html
<div class="hidden-xs col-md-1">(empty)</div>
<div class="col-xs-12 col-md-2"> ... </div>
```

would then become something like

```html
<div class="col-xs-12 col-md-2 col-md-push-1"> ... </div>
```

**Comment by newick** [comment]
@dedandy I found some empty divs in for example "methodology.html", can I help you on this one ?

**Comment by oncletom** [comment]
Addressed by #215 and #255
